doc/kernel/other/ring_buffers.rst

```diff
@@ -144,6 +194,42 @@ can be specified.
 	...
     }
 
+Bytes are copied to a ring buffer by calling
```

Comment:
For clarity change this to:

    Bytes are copied to a **byte mode** ring buffer by calling